test/commands/certs/add.js

```diff
@@ -633,6 +634,79 @@ biz.example.com  CNAME        biz.example.com.herokudns.com
       })
     })
 
+    it('# does not prompt if no domains and wildcard cert', function () {
+      let mock = nock('https://api.heroku.com')
+        .post('/apps/example/sni-endpoints', {
+          certificate_chain: 'pem content', private_key: 'key content'
+        })
+        .reply(200, endpointWildcard)
+
+      let domainsMock = nock('https://api.heroku.com')
+        .get('/apps/example/domains')
+        .reply(200, [])
+
+      return certs.run({app: 'example', args: {CRT: 'pem_file', KEY: 'key_file'}, flags: {bypass: true}}).then(function () {
+        mock.done()
+        domainsMock.done()
+        expect(unwrap(cli.stderr)).to.equal('Adding SSL certificate to example... done\n')
+        /* eslint-disable no-trailing-spaces */
+        /* eslint-disable no-irregular-whitespace */
+        expect(cli.stdout).to.equal(
+          `Certificate details:
+Common Name(s): *.example.org
+Expires At:     2013-08-01 21:34 UTC
+Issuer:         /C=US/ST=California/L=San Francisco/O=Heroku by Salesforce/CN=secure.example.org
+Starts At:      2012-08-01 21:34 UTC
+Subject:        /C=US/ST=California/L=San Francisco/O=Heroku by Salesforce/CN=secure.example.org
+SSL certificate is self signed.
+
+=== Your certificate has been added successfully.  Add a custom domain to your app by running heroku domains:add <yourdomain.com>
+`)
+        /* eslint-enable no-irregular-whitespace */
+        /* eslint-enable no-trailing-spaces */
+      })
+    })
+
```

Comment:
Also, are you interested in adding a test for correct domains(no warnings) + wildcard cert?